client/webpack.config.js

```diff
@@ -49,6 +50,7 @@ module.exports = {
         ]
     },
     plugins: [
-        new HtmlWebpackPlugin({ template: path.resolve(__dirname, 'src/index.html') })
+        new HtmlWebpackPlugin({ template: path.resolve(__dirname, 'src/index.html') }),
+        new CopyWebPackPlugin([{from:'src/images',to:'images'}])
```

Comment:
Does this actually copy the images to `client/dist/images` folder?  That's where server will be serving static files from.

Reply:
Just tried it out and it works perfectly.